subprojects/instant-execution/src/integTest/groovy/org/gradle/instantexecution/InstantExecutionProblemReportingIntegrationTest.groovy

```diff
@@ -52,6 +52,7 @@ class InstantExecutionProblemReportingIntegrationTest extends AbstractInstantExe
 
         and:
         instant.assertStateStoreFailed()
+        outputContains("Configuration cache entry not stored.")
```

Comment:
❌ Let's use a different verb instead of _not stored_ which it's very easily mistaken by _stored_ in a wall of text, maybe _discarded_? _Configuration cache entry discarded._

Reply:
Good point! Done